File: python/raindrops/raindrops.py
def convert(number):
    if(number % 3 == 0):
        if(number % 5 == 0):
            if(number % 7 == 0):
                return 'PlingPlangPlong'
            return 'PlingPlang'
        elif(number % 7 == 0):
            return 'PlingPlong'
        return 'Pling'
    elif(number %5 == 0):
        if(number % 7 == 0):
            return 'PlangPlong'
        return 'Plang'
    elif(number % 7 == 0):
        return 'Plong'
    else:
        return str(number)

File: go/raindrops/raindrops.go
//Package raindrops implements the Convert method
package raindrops

//Convert takes in an integer and returns some combination
//of Pling Plang Plong, depending if the integer is divisible
//by 3, 5, and 7 respectivly.
func Convert(a int) string {
	rain := ""

	if a%3 == 0 {
		rain += "Pling"
	}
	if a%5 == 0 {
		rain += "Plang"
	}
	if a%7 == 0 {
		rain += "Plong"
	}

	return rain

}
